db/snapshot_test.go

```diff
@@ -33,18 +33,28 @@ func TestSnapshot(t *testing.T) {
 	require.NoError(t, err)
 	defer snapshot.Release()
 
-	// Any models we add after taking the snapshot should not be included in the
-	// query.
+	// Any models we add after taking the snapshot should not affect the query.
 	for i := 0; i < 5; i++ {
 		model := &testModel{
 			Name: "OtherPerson_" + strconv.Itoa(i),
-			Age:  i,
+			Age:  42,
```

Comment:
There was also a bug in the tests. These models weren't showing up in the query results because the query is specifically looking for models where `Age == 42`.